Contribute/index.md

```diff
@@ -14,32 +14,54 @@ ms.custom: external-contributor-guide
 
 Welcome to the [docs.microsoft.com](https://docs.microsoft.com) (Docs) Contributor Guide!
 
-## About this guide
+Several of our documentation sets are open source, hosted on GitHub. More teams are adopting this model all the time. Even document sets that are not completely open source have public-facing repos where you are invited to make pull requests. It streamlines and improves communication between the product engineers, the content teams, and our customers. Working in the open provides several advantages:
 
-Here you'll find all of the information you need to contribute to Docs articles, by using the Microsoft Open Publishing Services (OPS) platform and the supporting tools and processes.
+- Open source repos plan in the open to get feedback on what docs are most needed.
+- Open source repos review in the open to publish the most helpful content on our first release.
+- Open source repos update in the open to make it easier to continuously improve the content.
 
-The table of contents to your left is designed to help you get started and be productive contributing to Microsoft Docs. The introductory articles provide a quick start for the tasks that are common to any contribution activity. Later articles are specific to different tasks, and you should focus on the section that describes the activity that interests you. Many of the articles can also serve as reference content, which you might want to save as a favorite/bookmark in your browser. Also note that there are several links to other sites, which will take you to pages that are off the docs.microsoft.com domain and outside this guide.
+The user experience on [docs.microsoft.com](https://docs.microsoft.com) integrates [GitHub](https://github.com) workflows directly to make it even easier. Start by [editing the document you are viewing](#quick-edits-to-existing-documents). Or, help by [reviewing new topics](#review-open-prs), or [create quality issues](#create-quality-issues).
 
-## Contribution tasks
+> [!IMPORTANT]
+> All repositories that publish to docs.microsoft.com have adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/) or the [.NET Foundation Code of Conduct](https://dotnetfoundation.org/code-of-conduct). For more information, see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/). Or contact [opencode@microsoft.com](mailto:opencode@microsoft.com), or [conduct@dotnetfoundation.org](mailto:conduct@dotnetfoundation.org) with any questions or comments.<br>
+>
+> Minor corrections or clarifications to documentation and code examples in public repositories are covered by the [docs.microsoft.com Terms of Use](https://docs.microsoft.com/legal/termsofuse). New or significant changes generate a comment in the pull request, asking you to submit an online Contribution License Agreement (CLA) if you are not an employee of Microsoft. We need you to complete the online form before we can accept your pull request.
```

Comment:
The online CLA form must be completed before we can review your PR.

We shouldn't be reviewing a PR without a signed agreement.